### ISSUE TITLE
[COGNITION, SCREENREADER]: Dates SHOULD be spelled out, as documented in the VA design system

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8520]
#  [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The [VA design system](https://design.va.gov/content-style-guide/dates-and-numbers#dates-and-years) specifies dates **should** spell out the month. 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to understand dates on a page.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter `<IP ADDRESS>` in browser
2. Start screenreading device listed in Environment
3. Read the content of the page
4. Verify the date is read numerically

## Possible Fixes (optional)

Convert the date provided by the database into Month DD, YYYY format

## WCAG or Vendor Guidance (optional)

* [VA design system](https://design.va.gov/content-style-guide/dates-and-numbers#dates-and-years) 

## Screenshots or Trace Logs

![Screen Shot 2020-04-28 at 2 07 04 PM](https://user-images.githubusercontent.com/57469/80521899-c34e3a80-8959-11ea-83ca-16a6d156e2ab.png)

**Comment by jason-gcio** [comment]
05/01/1955 should be May 1, 1955

**Comment by jenstrickland** [comment]
@micahchiang I'm sorry, I need to validate before we close these tickets.

**Comment by micahchiang** [comment]
@jenstrickland - sounds good. The pull request the corresponds to this ticket and #8515 has been merged, so the updates should be on staging either later today or Monday.

**Comment by jason-gcio** [comment]
@jenstrickland - User 1 on this url should get us to where we need to be for this ticket as well: https://staging.va.gov/view-change-dependents/view

**Comment by jenstrickland** [comment]
@jason-gcio I'm seeing ALL-CAPS for the name. Is this due to the data in the database or was this a styling decision?




![Screen Shot 2020-07-30 at 4 27 50 PM](https://user-images.githubusercontent.com/57469/88971222-a17a9180-d281-11ea-9fb3-1ba8365cad91.png)

**Comment by jenstrickland** [comment]
For this specific ticket, though, validated that the dates are indeed spelled out.

**Comment by jason-gcio** [comment]
We manipulate data as it comes in as little as possible...def not a decision on our end.
